sjb/config/test_cases/test_pull_request_crio_ami_rhel.yml

```diff
@@ -1,7 +1,6 @@
 ---
 parent: 'common/test_cases/crio.yml'
 overrides:
-  timer: 'H H * * *'
```

Comment:
This job is already a postsubmit:

https://github.com/openshift/release/blob/414148d7e853423b110122ddf9ff77aeee0ff265/cluster/ci/config/prow/config.yaml#L583-L590

I'm assuming it does not need to be periodic too.

Reply:
Presubmit but I think it's fine.